effekt/jvm/src/main/scala/effekt/EffektConfig.scala

```diff
@@ -58,6 +58,8 @@ class EffektConfig(args: Seq[String]) extends REPLConfig(args) {
     group = common
   ).map(Backend.backend)
 
+  def runArgs(): Seq[String] = args.dropWhile(_ != "--").drop(1)
```

Comment:
I first thought this is gonna crash at runtime for arguments that do not contain `--`, but I wasn't aware of the contract on `drop`.